### ISSUE TITLE
cgi is deprecated and slated for removal in Python 3.13

### ISSUE THREAD

**Issue by ldeluigi** [jyksnw/install-jdk#97]
```
..\..\..\AppData\Python311\Lib\site-packages\jdk\client\client.py:1
  D:\AppData\Python311\Lib\site-packages\jdk\client\client.py:1: DeprecationWarning: 'cgi' is deprecated and slated for removal in Python 3.13
    import cgi
```

**Comment by jyksnw** [comment]
@ldeluigi, thanks for reporting this. I will start looking for a replacement solution.

**Comment by ldeluigi** [comment]
News?

**Comment by jyksnw** [comment]
This is still planned for a future release of the library. This library aims to target active, released Python versions greater than Python 3.6. Python version 3.13 is scheduled for release in Oct. 2024, the plan is to have this deprecated usage addressed prior to Python 3.13 release. If you have a more immediate reason for this fix than please tell me the use case or issue so that I can properly prioritize the issue.

**Comment by ldeluigi** [comment]
Thanks for your answer and your work. It's not mandatory nor a high priority issue, but this raises a big yellow warning every time I run tests. I'm fine with it anyway, trusting it will be fixed before 3.13, thanks again.

**Comment by jyksnw** [comment]
@ldeluigi I was able to replace the CGI module along with a few other deprecated methods from the library. This should clear your warnings when you run tests.
